### PR TITLE
Files for Operating Systems at TTPU

### PR DESCRIPTION
**Pull request by frisso**

### CODE REVIEW

File: provisioning/virtual-machines/ansible/ttpu-os/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# tasks file for ttpu-os
+
+- name: Install generic utility packages
+  apt:
+    name: "{{ to_install }}"
+    state: present
+  vars:
+    to_install:
+    - iperf3
+    - nginx
```
Comment (giorio94) [inline provisioning/virtual-machines/ansible/ttpu-os/tasks/main.yml:11]: Are these packages required?

**Review by giorio94 — APPROVED** [review]
/lgtm

File: provisioning/virtual-machines/ansible/ttpu-os/tasks/main.yml (hunk 1 above)
Comment (frisso) [reply provisioning/virtual-machines/ansible/ttpu-os/tasks/main.yml:11]: Yes, we use them in the lab.

File: provisioning/virtual-machines/ansible/ttpu-os/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+# tasks file for ttpu-os
+
+- name: Install generic utility packages
```
Comment (frisso) [inline provisioning/virtual-machines/ansible/ttpu-os/tasks/main.yml:4]:
```suggestion
- name: Install additional software used in lab
```

**Review by palexster — APPROVED** [review]

**Pull request merged**